src/app/fdctl/run/tiles/dedup.c

```diff
@@ -12,6 +13,7 @@ init( fd_tile_args_t * args ) {
   /* calling fd_tempo_tick_per_ns requires nanosleep, it is cached with
      a FD_ONCE */
   fd_tempo_tick_per_ns( NULL );
+  metrics_boot( args->metrics_pod, metrics_verify, args->tile_idx );
```

Comment:
Howcome this is happening in init? Might be poorly named but init is for privileged setup, and couldn't see anything privileged in there.

Reply:
Updated to be in `run()`